### ISSUE TITLE
Sniper Elite 5 DLC Incomplete Interactive Maps

### ISSUE THREAD

**Issue by MTMabowels** [lordfiSh/sniperelite5maps#19]
I note your interactive maps are empty for all the DLC maps after Wolf Mountain.  Is this because you have not got around to it yet, or have you merely abandoned the project?

It’s no biggie to me either way but, as I am playing the game through again and I have Share Factory installed on my PS4, I could have a look around for you. If you have no objections that is. My thumb joint permitting of course, and when I remember to obviously.

**Comment by AtomCrafty** [comment]
Yeah, we just don't have the kind of free time we used to. We started populating the first DLC map a couple of months ago, but at the moment we spend the little gaming time we have on other games. We still plan on finishing the other maps eventually, but it's going to take a while.

Unfortunately the process of entering new markers isn't super intuitive and requires some knowledge of how the map data files are parsed. So I appreciate the offer of looking for items, but without the ability to enter them into the system it probably won't be of much use.

**Comment by MTMabowels** [comment]
Fair enough.   🤙.On 27 Oct 2024, at 09:34, AtomCrafty ***@***.***> wrote:﻿
Yeah, we just don't have the kind of free time we used to. We started populating the first DLC map a couple of months ago, but at the moment we spend the little gaming time we have on other games. We still plan on finishing the other maps eventually, but it's going to take a while.
Unfortunately the process of entering new markers isn't super intuitive and requires some knowledge of how the map data files are parsed. So I appreciate the offer of looking for items, but without the ability to enter them into the system it probably won't be of much use.
